src/Automation/Interfaces/IAxeWindowsActions.cs

```diff
@@ -49,5 +49,12 @@ internal interface IAxeWindowsActions
         /// </summary>
         /// <param name="path"></param>
         void RegisterCustomUIAPropertiesFromConfig(string path);
+
+        /// <summary>
+        /// Specify how Chromium content should be evaluated
+        /// </summary>
+        /// <param name="shouldTestAllChromiumContent">This can be set to true allow browser teams to test and debug code that converts from HTML to UIA.
```

Comment:
```suggestion
        /// <param name="shouldTestAllChromiumContent">This can be set to true to allow browser teams to test and debug code that converts from HTML to UIA.
```

Reply:
Fixed